### DISCUSSION TITLE
Failed to convert model  Error

### DISCUSSION THREAD

**Discussion by DenRakEiw** [smy20011/dreambooth-gui#90] [Q&A]
![image](https://user-images.githubusercontent.com/89697885/210223347-853191a3-2be8-41ad-90ae-f6834a763319.png)

Hello, I always get this Error.
I try to load the Model from Hugginface and also the way with the local Storrage.
I have an Nvida Quadro Card and Windows 11, Docker is running the Image.

**Comment by smy20011** [comment]
It seems the It's failed to train. Do you have the output in the output textbox?
